### ISSUE TITLE
Crashes on new tzinfo?

### ISSUE THREAD

**Issue by jyurek** [lau/tzdata#40]
We're getting this error intermittently and so we're assuming it's part of the autoupdate procedure that's crashing:

```
** Reason for termination ==
** {badarg,
       [{ets,lookup,[tzdata_current_release,release_version],[]},
        {'Elixir.Tzdata.ReleaseReader',current_release_from_table,0,
            [{file,"lib/tzdata/release_reader.ex"},{line,47}]},
        {'Elixir.Tzdata.ReleaseReader',simple_lookup,1,
            [{file,"lib/tzdata/release_reader.ex"},{line,14}]},
        {'Elixir.Tzdata.ReleaseReader',zone_and_link_list,0,
            [{file,"lib/tzdata/release_reader.ex"},{line,7}]},
        {'Elixir.Tzdata','zone_exists?',1,[{file,"lib/tzdata.ex"},{line,40}]},
        {'Elixir.Timex.Timezone',name_of,1,
            [{file,"lib/timezone/timezone.ex"},{line,142}]},
        {'Elixir.Timex.Timezone',do_get,3,
            [{file,"lib/timezone/timezone.ex"},{line,205}]},
        {'Elixir.Timex.Timezone',convert,2,
            [{file,"lib/timezone/timezone.ex"},{line,458}]}]}
{"Kernel pid terminated",application_controller,"{application_terminated,rtr,shutdown}"}
```

We have tzdata 0.5.12 and the tzdata version is currently 2017b on my local machine.

**Comment by OvermindDL1** [comment]
I 'was' experiencing the same thing for many days, but it seems to have stopped and is updating properly now without ever rebooting the server or anything...  o.O?

**Comment by lau** [comment]
Are you both using Timex? Is this happening during starting Timex / or you app? Or during compilation?

**Comment by jyurek** [comment]
We're using Timex, yes. This happens while the app is running and (typically) has been running for a while.

**Comment by OvermindDL1** [comment]
Yeah it happened just during the daily updates, the server had been up for weeks without restart, then all of a sudden it was working again after a few many days of it not working.  Been fine since, but definitely odd that it happened.

**Comment by MiLk** [comment]
We are experiencing the same issue.
It seems `Tzdata.ReleaseUpdater` is trying to create an ets table, but the table is already present and owner by `Tzdata.EtsHolder`.

**Comment by lau** [comment]
@Nicd & @MiLk the just released version `0.5.13` should fix the problem with `** (ArgumentError) argument error
    (stdlib) :ets.new(:tzdata_rel_2017c, [:set, :named_table])`
